Library/Formula/sfml.rb

```diff
@@ -27,9 +17,44 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Patch the bundled libs wording that causes linking errors. Accepted upstream.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  # https://github.com/ludvichzme/SFML/commit/c3f4f62f306f6f9b979b8c0cb2bfb3968712c8fa
+  patch :DATA
```

Comment:
Is it that the upstream commit cannot apply here or that we want a different patch for some reason? That's what I'm trying to understand. Also, will we still need a patch for the next release?